docs/source/roles/system_manager/manage_users.md

```diff
@@ -183,6 +183,18 @@ A sample email might look like the following
 >
 > --details about network and location/VPN restrictions here--
 
+(modifying_network)=
+
+## {{globe_with_meridians}} Manually modifying network restrictions
+
+One of the controls used by Tier 2/3 SREs is to restrict access based on network addresses. If the users were to require access from different IP addresses - if their institutional IP address changes, or an additional location with a different network were approved - then the network addresses that are allowed access can be modified manually.
+
+- In the Azure Portal, navigate to `RG_SHM_<SHM_ID>_SRE_<SRE_ID>_NETWORKING`
+- On the `Overview` tab, navigate to the Network Security Group for the appropriate remote desktop service (e.g `NSG_SHM_<SHM_ID>_SRE_<SRE_ID>_GUACAMOLE`)
```

Comment:
Is `NSG_SHM_<SHM_ID>_SRE_<SRE_ID>_GUACAMOLE` an example, or is that always the form of the NSG's name? (After removing MSRDS)

Reply:
I think we can cross that bridge when/if we come to it.

There is no sense in making the develop branch 'worse' or 'incorrect' in anticipation of merging the changes somewhere else later.

Reply:
```suggestion
- On the `Overview` tab, navigate to `NSG_SHM_<SHM_ID>_SRE_<SRE_ID>_GUACAMOLE`, the Network Security Group for the remote desktop service
```